ros_gz_bridge/src/convert/tf2_msgs.cpp

```diff
@@ -44,9 +44,15 @@ convert_gz_to_ros(
   tf2_msgs::msg::TFMessage & ros_msg)
 {
   ros_msg.transforms.clear();
+  builtin_interfaces::msg::Time ros_stamp;
+  convert_gz_to_ros(gz_msg.header().stamp(), ros_stamp);
+  
   for (auto const & p : gz_msg.pose()) {
     geometry_msgs::msg::TransformStamped tf;
     convert_gz_to_ros(p, tf);
+    tf.header.stamp = ros_stamp;
+    tf.header.frame_id = "world";
```

Comment:
not sure about this, maybe we should be able to configure this with a parameter, @azeey or @mjcarroll thoughts?